zad6/app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CustomersController < ApplicationController
+  def index
+    @customers = Customer.all
+  end
+  def show
```

Comment:
Zgodnie z zadanie, produkty należące do customera powinny być wyświetlane pod adresem `/customers/:customer_id/products`. Teraz jest to `/customers/:customer_id`. Zauważ, że ten `show` jest trochę mylący. Oczekiwalibysmy, że dostaniemy tutaj profil pojedynczego customera a dostajemy produkty 🤔 Żeby to poprawić należałoby:
* zanestować odpowiednio ścieżki
```ruby
# config/routes.rb

resources :customers, only: :index do
  resources :products, only: :index
end
```

Taki zapis rzeczywiście tworzy ścieżkę `/customers/:customer_id/products`, która odpowiada akcji `index` w kontrolerze `ProductsController`, czyli wszystko jest na swoim miejscu - `CustomersController` zajmuje się customerami a `ProductsController` produktami. 😉